src/Test/L0/TestHostContext.cs

```diff
@@ -34,13 +34,15 @@ public TestHostContext(object testClass, [CallerMemberName] string testName = ""
             _suiteName = _suiteName.Replace(".", "_");
 
             // Setup the trace manager.
-            string traceFileName = $"trace_{_suiteName}_{_testName}.log";
-            if (File.Exists(traceFileName))
+            TraceFileName = Path.Combine(
+                IOUtil.GetBinPath(),
+                $"trace_{_suiteName}_{_testName}.log");
+            if (File.Exists(TraceFileName))
             {
-                File.Delete(traceFileName);
+                File.Delete(TraceFileName);
             }
 
-            Stream logFile = File.Create(traceFileName);
+            Stream logFile = File.Create(TraceFileName);
```

Comment:
This is for the loc string tests so that the assertion (on failure) can print the full log file name. That way, it's easier for the user to find the trace log with the missing/bad strings.